homeassistant/components/decora_wifi/strings.json

```diff
@@ -0,0 +1,22 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "title": "Setup Decora Wifi",
+        "description": "Your my.leviton.com credentials",
+        "data": {
+          "username": "Username",
+          "password": "Password"
```

Comment:
```suggestion
          "username": "[%key:common::config_flow::data::username%]",
          "password": "[%key:common::config_flow::data::password%]"
```

Reply:
ty fixed in https://github.com/home-assistant/core/pull/104573/commits/0566f258bb2bd99a7231282c6f55c0e937d508dc